--- a/meson.build	2021-10-01 11:42:17.000000000 +0000
+++ b/meson.build	2021-10-01 12:16:41.000000000 +0000
@@ -738,6 +738,7 @@
   # FIXME: check for readline
   # AX_LIB_READLINE
   readline_dep = declare_dependency(link_args: '-lreadline')
+  ncurses_dep = declare_dependency(link_args: '-lncurses')
   '''
   foreach readline_lib: ['-lreadline', '-ledit', '-leditline']
     if not is_variable('readline_dep')
--- a/src/nmcli/meson.build	2021-09-22 09:31:47.000000000 +0000
+++ b/src/nmcli/meson.build	2021-10-01 12:19:53.000000000 +0000
@@ -27,6 +27,7 @@
     libnm_dep,
     glib_dep,
     readline_dep,
+	ncurses_dep,
   ],
   link_with: [
     libnmc_setting,
